client/src/components/Picture/ProfilePic.js

```diff
@@ -13,25 +16,38 @@ const InitialDiv = styled.div`
   width: 7rem;
   text-align: center;
   font-weight: 500;
-  background-color: #f3f4fe;
+  background-color: ${(props) =>
+    props.hasPhoto ? colors.royalBlue : `#f3f4fe`};
   @media screen and (min-width: ${(props) =>
       props.resolution ? props.resolution : mq.tablet.narrow.minWidth}) {
     margin: 0;
     height: 80%;
-    line-height: 11rem;
-    width: 12rem;
+    line-height: ${(props) => (props.hasPhoto ? `10rem` : `11rem`)};
+    width: ${(props) => (props.hasPhoto ? `12.2rem` : `12rem`)};
     margin-right: 3rem;
     font-size: 5rem;
   }
 `;
-export default ({ noPic, initials, resolution }) => {
+
+const ProfilePic = ({ initials, resolution, user }) => {
   return (
-    <>
-      {noPic ? (
-        <InitialDiv resolution={resolution}>{initials}</InitialDiv>
+    <InitialDiv
+      resolution={resolution}
+      hasPhoto={user && user.photo ? true : false}
```

Comment:
Minor: unsure if this is required for React, but I think a ternary is unnecessary since `user && user.photo` will evaluate to a boolean

Reply:
AH YES